includes/admin/helper/views/html-oauth-start.php

```diff
@@ -0,0 +1,20 @@
+<?php defined( 'ABSPATH' ) or exit(); ?>
+
+<div class="wrap wc-helper">
+	<h1><?php _e( 'WooCommerce Helper', 'woocommerce' ); ?></h1>
+	<?php include( WC_Helper::get_view_filename( 'html-section-notices.php' ) ); ?>
+
+		<div class="start-container">
+			<div class="text">
+				<img src="https://woocommerce.com/wp-content/themes/woomattic/images/logo-woocommerce@2x.png" alt="WooCommerce" style="width:180px;">
+
+				<?php if ( ! empty( $_GET['wc-helper-status'] ) && $_GET['wc-helper-status'] = 'helper-disconnected' ) : ?>
```

Comment:
`$_GET['wc-helper-status'] = 'helper-disconnected'`

you're setting GET to `'helper-disconnected'` here.